src/index.html

```diff
@@ -11,7 +11,163 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
```

Comment:
It seems that the `<h1>` tag with the form heading is missing. Consider adding it back, e.g. `<h1>HTML Form</h1>` 📝